superset-frontend/plugins/plugin-chart-pivot-table/src/plugin/controlPanel.tsx

```diff
@@ -90,15 +89,40 @@ const config: ControlPanelConfig = {
         ],
         ['adhoc_filters'],
         emitFilterControl,
+        ['limit'],
```

Comment:
oh, also prefer the use of `series_limit` over `limit`

Reply:
👍